File: spring-boot-2-data-rest/src/main/java/com/example/springboot2rest/model/City.java
package com.example.springboot2rest.model;

import javax.persistence.Column;
import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;
import javax.persistence.Table;

import lombok.Data;

@Data
@Entity
@Table(name = "Municipio")
public class City {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column(name = "Id")
    private int id;

    @Column(name = "Codigo")
    private Integer cod;

    @Column(name = "Nome")
    private String name;

    @Column(name = "Uf" )
    private String uf;

}

File: spring-boot-webmvc-redis-cache/src/main/kotlin/com/github/andrelugomes/model/City.kt
package com.github.andrelugomes.model

import org.springframework.data.annotation.PersistenceConstructor
import java.io.Serializable
import javax.persistence.*

@Entity
@Table(name = "Municipio")
data class City(
    @Id
    @Column(name="Codigo")
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    var code: Int,

    @Column(name="Nome")
    var name: String,

    @Column(name="Uf")
    var uf: Int
): Serializable {
    constructor(): this(0,"", 0) //for serialization
}